theano/gof/cc.py

```diff
@@ -1282,26 +1282,19 @@ def in_sig(i, topological_pos, i_idx):
                 return ((), sig)
         return version, sig
 
-    def compile_cmodule(self, location=None):
-        """
-        Compile the module and return it.
+    def get_src_code(self):
```

Comment:
This method isn't used and cause problem as it didn't modify the mod as done around line 1306